lib/teselagen/api/tests/test_test_client.py

```diff
@@ -323,13 +338,16 @@ def test_get_assays(
                    for key in ["id", "name"])
         # assert all( element[key] is not None for element in response for key in ["id", "name"])
 
-    # @pytest.mark.skip(reason="We should expect an 'assay' key that may be None.")
+    # @pytest.mark.skip(reason="The files endpoints are under maintenance.")
     def test_get_files(
         self,
-        logged_client: TESTClient,
+        logged_client: TeselaGenClient,
+        host_url: str,
         select_laboratory,
     ) -> None:
-        client = logged_client
+        if 'platform' in host_url:
+            return
```

Comment:
Will these tests be successful even if the code is not executed? If this is so, the test results could be "inflated" (successful tests of untested code)

Maybe we could install [pytest lazy fixture](https://pypi.org/project/pytest-lazy-fixture/) to be able to use `fixtures` as `parameters` in `@pytest.mark.parametrize`,  so, we could use `fixtures` in `@pytest.mark.parametrize`, to skip some tests when `"platform" in host_url` by passing it a `pytest.mark.skipif`, which is documented [here](https://docs.pytest.org/en/6.2.x/skipping.html#skip-xfail-with-parametrize)